rllib/core/models/tf/base.py

```diff
@@ -0,0 +1,65 @@
+import abc
```

Comment:
Moved this from another file. Now this is analogous to the already existing `torch/base.py`.
Both define the DL-specific base RLlib Model classes TfModel and TorchModel.